css/main.css

```diff
@@ -458,6 +461,100 @@ body {
   margin: 18px 0;
 }
 
+.modal {
+  background: #fff;
+  padding: 1.5rem;
+  width: 90%;
+  opacity: 0;
+  visibility: hidden;
+  transition: transform 1s, opacity 1s, visibility 0s;
+  z-index: 300;
+}
+
+.modal__header {
+  justify-content: space-between;
+}
+
+.modal__header img {
+  width: 20px;
+  height: 20px;
+  cursor: pointer;
+}
+
+.modal__image {
+  width: 100%;
+  margin: 1rem 0;
+}
+
+.modal__dsc__second {
+  flex-direction: column;
+  align-items: flex-start;
+  flex-basis: 40%;
+}
+
+.modal__dsc__second div {
+  margin: 1rem 0;
+}
+
+.modal__dsc__second p {
+  padding: 4px 12px;
+  border-radius: 8px;
+  margin-right: 0.5rem;
+}
+
+.modal__btn {
+  margin: 1rem 0;
+}
```

Comment:
- [ ] Please make sure to place appropriate icons inside the model buttons and it matches the design.

### Your design

![image](https://user-images.githubusercontent.com/83803414/167876444-14359321-931f-41f0-97ef-9fca7f884143.png)

### Figma design

![image](https://user-images.githubusercontent.com/83803414/167876518-5afe25c1-9f82-45ec-b94d-8a56faaa11a9.png)

Reply:
ok